clouddriver-kubernetes/clouddriver-kubernetes.gradle

```diff
@@ -52,6 +52,7 @@ dependencies {
   implementation project(":clouddriver-api")
   implementation project(":clouddriver-artifacts")
   implementation project(":clouddriver-core")
+  implementation project(":clouddriver-configserver")
   implementation project(":cats:cats-core")
   implementation project(":clouddriver-security")
```

Comment:
I think you can remove json-flattener from here.

Reply:
I get a gradle build error if I remove that

Reply:
Sorry, too many working directories...me too.